docs/api-reference/web/adapters/sollet.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: sollet-adapter
+hide_table_of_contents: false
+displayed_sidebar: apiReference
+---
+
+# Sollet Adapter
+
+---
+
+import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs";
+
+## [`@web3auth/sollet-adapter`](https://npmjs.com/package/@web3auth/sollet-adapter)
+
+Sollet adapter allows you to connect with sollet wallet. You can read more about sollet wallet
+here.(https://github.com/project-serum/spl-token-wallet).
+
+## Basic Details
+
+---
+
+#### Adapter Name: `sollet`
+
+#### Package Name: [`@web3auth/sollet-adapter`](https://npmjs.com/package/@web3auth/sollet-adapter)
+
+#### authMode: `DAPP`
+
+#### chainNamespace: `SOLANA`
+
+#### Default: `YES`
+
+## Installation
+
+---
+
+<Tabs
+  defaultValue="npm"
+  values={[
+    { label: "npm", value: "npm" },
+    { label: "yarn", value: "yarn" },
+  ]}
+>
+
+<TabItem value="npm">
+
+```shell
+npm install --save @web3auth/sollet-adapter
+```
+
+</TabItem>
+
+<TabItem value="yarn">
+
+```shell
+yarn add @web3auth/sollet-adapter
+```
+
+</TabItem>
+
+</Tabs>
+
+## `SolletWebAdapter`
+
+---
+
+### Arguments
+
+<Tabs
+  defaultValue="table"
+  values={[
+    { label: "Table", value: "table" },
+    { label: "Interface", value: "interface" },
+  ]}
+>
+
+<TabItem value="table">
+
+| Parameter      | type                |
+| -------------- | ------------------- |
+| `chainConfig?` | `CustomChainConfig` |
+| `provider?`    | `string`            |
+
+</TabItem>
+
+<TabItem value="interface">
+
+```ts
+export declare class SolletWebAdapter extends BaseSolletAdapter {
+  readonly name: string;
+  constructor({ provider, ...options }?: SolletAdapterOptions);
+}
+export interface SolletAdapterOptions {
+  chainConfig?: CustomChainConfig;
+  provider?: string;
+}
+export declare class BaseSolletAdapter extends BaseAdapter<void> {
```

Comment:
why these interfaces are available in this adapter but on in others?

Reply:
I will add everywhere